packages/snap/src/index.tsx

```
@@ -52,11 +56,12 @@ export const onUserInput: OnUserInputHandler = async ({ id, event }) => {
 
       case 'advanced_options':
         // eslint-disable-next-line no-case-declarations
-        const currentTo = await StateManager.get('currentTo');
+        const currentStorageKey = await StateManager.get('currentStorageKey');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)